packages/providers/src.ts/fallback-provider.ts

```diff
@@ -365,17 +362,12 @@ async function getRunner(config: RunningConfig, currentBlockNumber: number, meth
             if (params.blockTag && isHexString(params.blockTag)) {
                 provider = await waitForSync(config, currentBlockNumber)
             }
-            return provider[method](params.address, params.blockTag || "latest");
+            return logger.throwError("NOT_SUPPORTED", Logger.errors.UNSUPPORTED_OPERATION);
         case "getStorageAt":
             if (params.blockTag && isHexString(params.blockTag)) {
                 provider = await waitForSync(config, currentBlockNumber)
             }
-            return provider.getStorageAt(params.address, params.position, params.blockTag || "latest");
-        case "getBlock":
```

Comment:
We should remove those 2 methods since we do not have them in the interface at all